menu.js

```diff
@@ -0,0 +1,11 @@
+const buttonMobile = document.getElementById('menu-toggle');
+const closingButton = document.getElementById('close');
+
+function toggleMenu() {
+  const nav = document.getElementById('nav');
+  nav.classList.toggle('menu-open');
+}
+
+buttonMobile.addEventListener('click', toggleMenu);
+closingButton.addEventListener('click', toggleMenu);
+document.querySelectorAll('.menu_links').forEach((item) => { const nav = document.getElementById('nav'); item.addEventListener('click', () => { nav.classList.remove('menu-open'); }); });
```

Comment:
[OPTIONAL]

- Awesome implementation of the mobile menu feature. Well done 👍 What do you think about keeping the codebase clean by separating the code on line 11into multiple lines? 

Here 👇  is a cleaner/more readable version of your code 😀 

![](https://user-images.githubusercontent.com/54848363/150879510-df74220c-6d63-414a-9d10-d2601b52aac5.png)

Reply:
Thank you for your excellent suggestion. I definitely will.